### DISCUSSION TITLE
Usage of the make_batch_for_iteratively_mapping_assembling.py script for genome assembly

### DISCUSSION THREAD

**Discussion by DenisGoryunov** [Kinggerm/GetOrganelle#206] [Q&A] [answered]
In the FAQ chapter of documentation make_batch_for_iteratively_mapping_assembling.py script was mentioned. Where can i find information  (e.g. input data, command line options, etc.) about usage of the script for genome assembly?

**Comment by Kinggerm** [comment, marked as answer]
The instruction of all scripts can be accessed by attaching `-h` or `--help` to the command, e.g.

    make_batch_for_iteratively_mapping_assembling.py -h

**Reply by DenisGoryunov** [reply]
Thanks a lot!
